packages/ui/src/icons/icon/create-icon.native.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { ComponentPropsWithRef, ElementRef, forwardRef } from 'react';
+import Svg, { SvgProps } from 'react-native-svg';
+
+import { useTheme } from '@shopify/restyle';
+
+import { Theme } from '../../theme-native';
+
+type IconSize = 'small' | 'medium' | 'large';
```

Comment:
Can you introduce a `types` file and share the types between web and native?

Reply:
Good call. Re-used both this and the size map.